src/Evolution/Systems/CurvedScalarWave/CalculateGrVars.hpp

```diff
@@ -26,27 +29,38 @@ namespace CurvedScalarWave::Actions {
 /// \brief Action that initializes or updates items related to the
 /// spacetime background of the CurvedScalarWave system
 ///
+/// If `SkipForStaticBlocks` is `true`, then this action does nothing if the
+/// block of the domain is time independent. This is a performance optimization
+/// to avoid updating the background spacetime in blocks that are time
+/// independent. Note that this assumes that the background spacetime is also
+/// time independent.
+///
 /// DataBox changes:
 /// - Adds:
 ///   * `CurvedScalarWave::System::spacetime_tag_list`
 /// - Removes: nothing
 /// - Modifies: nothing
-
-template <typename System>
+template <typename System, bool SkipForStaticDomains = false>
```

Comment:
[optional] I've been burned badly before by having default template parameters that significantly change the behavior of the algorithm. Consider removing the default?

Reply:
ok done